developer-docs-site/docs/concepts/state-sync.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "State Synchronization"
+slug: "state-sync"
+---
+
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# State Synchronization
+
+Nodes in an Aptos network, both the validator nodes and the fullnodes, must always be synchronized to the latest Aptos blockchain state. The state synchronization (state sync) component that runs on each node is responsible for this synchronization. To achieve this synchronization, the state sync identifies and fetches new blockchain data from the peers, validates the data and persists it to the local storage.
+
+## State sync modes
+
+If you are a node operator, you can configure the state sync mode to operate in one of the two synchronization modes, depending on the state of the node, i.e., whether the node has just started (bootstrapping mode) or has been running for some time (continuous sync mode). 
```

Comment:
This is a little misleading. State sync always runs both modes and operators can't configure which one runs (i.e., all nodes will first bootstrap on startup, and then continuously sync, operators can't change this). What operators can configure is the internal details of these modes (as outlined below), but they won't need to "configure the state sync mode to operate in one of the two synchronization modes". Both modes are always used.